src/applications/model/gaming-streaming-server.cc

```diff
@@ -60,7 +64,10 @@ GamingStreamingServer::GamingStreamingServer ()
     m_totSentBytes (0),
     m_socket (0),
     m_peerAddress (Address ()),
-    m_peerPort (0)
+    m_peerPort (0),    
+    m_referenceBitRate (1),
+    m_scalingFactor (1)
```

Comment:
do not initialize if an attribute is supposed to initalize it instead. check all variables related to attributes, even the past ones